packages/core/src/components/button/button.tsx

```diff
@@ -85,7 +85,7 @@ export class Button implements IxButtonComponent {
   }
 
   private getAssociatedForm(): HTMLFormElement | null {
-    if (this.internals && 'form' in this.internals && this.internals.form) {
+    if (this.internals?.form) {
       return this.internals.form;
     }
     return this.hostElement.closest('form');
```

Comment:
`internals` are getting provided by stencil. If `attachAssociated: true` cannot be used due lack of browser api, the component will fail with an error.

Normally there is no other branch possible e.g `this.hostElement.closest('form')`